autoload/sy/repo.vim

```diff
@@ -365,20 +435,41 @@ endfunction
 " s:initialize_job {{{1
 function! s:initialize_job(vcs) abort
   let vcs_cmd = s:expand_cmd(a:vcs, g:signify_vcs_cmds)
+  return s:wrap_cmd(a:vcs, vcs_cmd)
+endfunction
+
+" Function: s:initialize_buffer_job {{{1
+function! s:initialize_buffer_job(vcs) abort
+  let bufferfile = tempname()
+  call s:write_buffer(bufferfile)
+
+  let basefile = tempname()
+  let base_cmd = s:get_base_cmd(a:vcs) . '>' . fnameescape(basefile) . ' && '
```

Comment:
I think this should work with cmd.exe, but fail with Powershell.. but that doesn't matter for now. If there's no other way, I'll simply force `cmd.exe` on Windows (like we do with `sh` on all other systems).